aws/sqs/src/main/scala/com/commercetools/queue/aws/sqs/SQSClient.scala

```diff
@@ -83,4 +83,12 @@ object SQSClient {
       }
       .map(new SQSClient(_))
 
+  /**
+   * Creates an SQS client from a given instance of `SqsAsyncClient`.
+   *
+   * It is upon the caller to ensure closing of the underlying client once appropriate.
+   */
+  def fromClient[F[_]](client: SqsAsyncClient)(implicit F: Async[F]): QueueClient[F] =
+    new SQSClient(client)
```

Comment:
Looking at what it does, I felt it'd be overkill to add a test by running the entire test suite for a client constructed through his method. But can change if you prefer.